File: DotNetWebAPI/DotNetWebAPI/DTOs/ProductToAdd.cs
namespace DotNetWebAPI.DTOs
{
    public class ProductToAdd
    {
        public int Id { get; set; }
        public string Title_EN { get; set; }
        public string Title_AR { get; set; }
        public string Details_EN { get; set; }
        public string Details_AR { get; set; }
        public decimal price { get; set; }

        public int Color { get; set; }
        public int Quantity { get; set; }
        public int? CategoryID { get; set; }
        public string? OwnerId { get; set; }
        public List<IFormFile>? files { get; set; }
    }
}

File: AngularProject/client/src/app/Models/Owner/ProductToAdd.ts
export class ProductToAdd {
    id?: number;
    title_EN!: string;
    title_AR!: string;
    details_EN!: string;
    details_AR!: string;
    price!: number;
    color: number = 1;
    quantity!: number;
    categoryID!: number;
    ownerId!: string;
    // ImagesFiles?: FormData = new FormData();
}
